README.md

```diff
@@ -99,23 +134,20 @@ Alternatively, you can use this command to start the server in development mode.
 npm run dev
 ```
 
-5. Once you have your server running, you need to expose your `localhost` to a public domain so the Twilio webhook can reach the expected endpoint. This is easy using `ngrok`:
+9. Once you have your server running, you need to expose your `localhost` to a public domain so the Twilio webhook can reach the expected endpoint. This is easy [using the Twilio CLI](https://www.twilio.com/docs/twilio-cli/general-usage#proxying-your-localhost).
+```
+twilio phone-numbers:update [PN sid or E.164] --sms-url http://localhost:3000/pay
 ```
-ngrok http 3000
-``` 
-This will generate a url similar to: `https://cd2ef758.ngrok.io`. Copy that link (you must use `https` or Twilio will reject the request).
-
-6. Now, you must take that public URL and configure this as a webhook for one of your phone numbers in the console. [Here is small a guide](https://www.twilio.com/docs/voice/quickstart/node#allow-twilio-to-talk-to-your-application) on how to that.
 
-**IMPORTANT!** Don't forget to add `/pay` (so it looks like `https://cd2ef758.ngrok.io/pay`) at the end of the url since that is the endpoint in which the application will listen for the call.
+This will generate a url similar to: `https://cd2ef758.ngrok.io`.
 
-7. Navigate to [http://localhost:3000](http://localhost:3000) to see some sample credit card details to test the payment.
+10. Navigate to [http://localhost:3000](http://localhost:3000) to see some sample credit card details to test the payment.
 
-8. You can also navigate to [http://localhost:3000/config](http://localhost:3000/config) to override the default payment details.
+11. You can also navigate to [http://localhost:3000/config](http://localhost:3000/config) to override the default payment details.
 
-9. That's it! Now call the Twilio phone number you configured and follow the instructions to complete the payment.
+12. That's it! Now call the Twilio phone number you configured and follow the instructions to complete the payment.
 
-10. You can see if the payment was charged on your Stripe dashboard. Take a look at [this](https://www.twilio.com/docs/voice/tutorials/how-capture-your-first-payment-using-pay#test-your-application) for more details.
+13. You can see if the payment was charged on your Stripe dashboard. Take a look at [this](https://www.twilio.com/docs/voice/tutorials/how-capture-your-first-payment-using-pay#test-your-application) for more details.
```

Comment:
The `[this]` link should be more descriptive for accessibility and SEO. Something like `[Take a look at this tutorial]`

Reply:
👍